salt/orch/update.sls

```diff
@@ -62,6 +62,7 @@ pre-orchestration-migration:
     - batch: 3
     - sls:
       - cni.update-pre-orchestration
```

Comment:
Can't we get rid of `cni.update-pre-orchestration`?

Reply:
We could combine them, but then CNI migration stuff would end up in the kubelet role, vs just having "move important data" there. Yes, that data happens to just be CNI data, but still - it felt cleaner.

Reply:
I agree, thank you.